docs/BACKEND-ROADMAP.md

```
@@ -355,6 +355,409 @@ export const env = envSchema.parse(process.env)
+// GET /api/study-sessions?month=2025-01
+// Retorna histórico do mês
+
+// GET /api/study-sessions/streak
+// Retorna dias consecutivos de estudo
+```
+
+---
+
+## 📅 Timeline Sugerido (Pós-TCU)
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)